common/containers/Tabs/GenerateWallet/components/PaperWallet.tsx

```diff
@@ -1,113 +1,91 @@
+import React from 'react';
 import { Link } from 'react-router-dom';
 import translate from 'translations';
 import './PaperWallet.scss';
 import Template from './Template';
 
-interface Props {
-  wallet: PrivKeyWallet;
-}
+const content = (wallet: IFullWallet) => (
+  <div className="GenPaper">
+    {/* Private Key */}
+    <h1 className="GenPaper-title">{translate('GEN_Label_5')}</h1>
+    <input
+      className="GenPaper-private form-control"
+      value={wallet.getPrivateKeyString()}
+      aria-label={translate('x_PrivKey')}
+      aria-describedby="x_PrivKeyDesc"
+      type="text"
+      readOnly={true}
+    />
 
-export default class PaperWallet extends Component<Props, {}> {
-  public render() {
-    const { wallet } = this.props;
+    {/* Download Paper Wallet */}
+    <h1 className="GenPaper-title">{translate('x_Print')}</h1>
+    <div className="GenPaper-paper">
+      <PrintableWallet wallet={wallet} />
+    </div>
 
-    const content = (
-      <div className="GenPaper">
-        {/* Private Key */}
-        <h1 className="GenPaper-title">{translate('GEN_Label_5')}</h1>
-        <input
-          className="GenPaper-private form-control"
-          value={wallet.getPrivateKey()}
-          aria-label={translate('x_PrivKey')}
-          aria-describedby="x_PrivKeyDesc"
-          type="text"
-          readOnly={true}
-        />
+    {/* Warning */}
+    <div className="GenPaper-warning">
+      <p>
+        <strong>Do not lose it!</strong> It cannot be recovered if you lose it.
+      </p>
+      <p>
+        <strong>Do not share it!</strong> Your funds will be stolen if you use
+        this file on a malicious/phishing site.
+      </p>
+      <p>
+        <strong>Make a backup!</strong> Secure it like the millions of dollars
+        it may one day be worth.
+      </p>
+    </div>
 
-        {/* Download Paper Wallet */}
-        <h1 className="GenPaper-title">{translate('x_Print')}</h1>
-        <div className="GenPaper-paper">
-          <PrintableWallet wallet={wallet} />
-        </div>
+    {/* Continue button */}
+    <Link className="GenPaper-continue btn btn-default" to="/view-wallet">
+      {translate('NAV_ViewWallet')} →
+    </Link>
+  </div>
+);
 
-        {/* Warning */}
-        <div className="GenPaper-warning">
-          <p>
-            <strong>Do not lose it!</strong> It cannot be recovered if you lose
-            it.
-          </p>
-          <p>
-            <strong>Do not share it!</strong> Your funds will be stolen if you
-            use this file on a malicious/phishing site.
-          </p>
-          <p>
-            <strong>Make a backup!</strong> Secure it like the millions of
-            dollars it may one day be worth.
-          </p>
-        </div>
+const help = (
+  <div>
+    <h4>{translate('GEN_Help_4')}</h4>
+    <ul>
+      <li>
+        <NewTabLink href="https://myetherwallet.groovehq.com/knowledge_base/topics/how-do-i-save-slash-backup-my-wallet">
```

Comment:
All `groovehq.com` links are out of date. Probably not necessary to fix them in this PR, just making note.

Reply:
Sounds good, i'll just have to pass the `n` parameter in `toV3Wallet` so it does less iterations. As for the keystore file address generation, i can probably just make a wrapper function for that